packages/cli/src/credentials/credentials.controller.ee.ts

```diff
@@ -190,6 +191,10 @@ EECredentialsController.put(
 			sharees_removed: amountRemoved,
 		});
 
+		const isEmailingEnabled = config.getEnv('userManagement.emails.mode') === 'smtp';
```

Comment:
This feels like a leaky abstraction. I think a better approach could be to update the `UserManagementMailer.notifyCredentialsShared` to accept `newShareeIds`, and let that class resolve ids to email, as well as let that class be the only place with this `isEmailSetUp` check.